tests/e2e/tests/web/critical-path/database/connecting-to-the-db.e2e.ts

```diff
@@ -181,3 +181,23 @@ test
         await myRedisDatabasePage.clickOnDBByName(sshDbPass.databaseName);
         await Common.checkURLContainsText('browser');
     });
+test
+    .meta({ rte: rte.none })
```

Comment:
shouldn't we update existing tests where we added db manually, if such tests exist?
and I am not sure about saving the default tab, will it be or not. I mean that if the previous executed tests were RDI tests will it affect tests, that the started tab will be RDI tab

Reply:
thanks, I'll check this

Reply:
updated